examples/gatsby/src/pages/shop.js

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react'
+import { Link, graphql, useStaticQuery } from 'gatsby'
+
+export default function AllProducts() {
+  const productsData = useStaticQuery(graphql`
```

Comment:
Is it possible to pull this out into a `query` variable like you've done in `collection.js`?

Reply:
?